monitor-manage-device-with-iotcentral/CoffeeMaker.js

```diff
@@ -0,0 +1,266 @@
+// Use the Azure IoT device SDK for devices that connect to Azure IoT Central.
+var iotHubTransport = require('azure-iot-device-mqtt').Mqtt;
+var Client = require('azure-iot-device').Client;
+var Message = require('azure-iot-device').Message;
+var ProvisioningTransport = require('azure-iot-provisioning-device-mqtt').Mqtt;
+var SymmetricKeySecurityClient = require('azure-iot-security-symmetric-key').SymmetricKeySecurityClient;
+var ProvisioningDeviceClient = require('azure-iot-provisioning-device').ProvisioningDeviceClient;
+
+// DPS connection information
+var modelIdObject = { modelId: 'dtmi:com:example:ConnectedCoffeeMaker;1' };
+var provisioningHost = 'global.azure-devices-provisioning.net';
+var idScope = '{ID scope}';
+var registrationId = '{Device ID}';
+var symmetricKey = '{Primary key}';
```

Comment:
Maybe use environment variables instead to discourage users from pasting a plaintext key into the sample

Reply:
This also simplifies the exercise for the user as they don't need to edit the code.